### PR TITLE
Product v2

### PR DESCRIPTION
**Pull request by dipen2000**
Please review the products feature of my E-comm app

### CODE REVIEW

File: src/Components/Filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* .filter-container{
```
Comment (Shivani-1524) [inline src/Components/Filter/Filter.css:1]: remove comments

File: src/Components/ProductListing/ProductListing.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+.video-section{
+    border : 3px solid green;
+    padding : 1rem;
+}
+
+.video-section-grid{
+    border : 3px solid purple;
+    display : grid;
+    justify-items: center;
+    grid-template-columns: repeat(auto-fill , minmax(26rem , 1fr));
```
Comment (Shivani-1524) [inline src/Components/ProductListing/ProductListing.css:24]: nice use of minmax

**Review by Shivani-1524 — APPROVED** [review]
do look into the suggestions ive given in your other PRs. they apply here too.

File: src/Pages/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      setToken(encodedToken);
+      localStorage.setItem("isAuth", true);
+      localStorage.setItem("token", encodedToken);
+      setInputData((prevData) => ({
+        ...prevData,
+        input: { email: "", password: "" },
+      }));
+      navigate("/");
+    } catch (e) {
+      console.error(e);
```
Comment (rahulyadav139) [inline src/Pages/Login/Login.js:54]: suggestion: instead of just console logging the error, show some error message.

**Review by rahulyadav139 — APPROVED** [review]

**Pull request merged**